### DISCUSSION TITLE
List of Custom filter?

### DISCUSSION THREAD

**Discussion by hlytus** [ReVanced/revanced-discussions#969] [Question]
Can you share a list of custom user filters. I only know _related_video_
![233563633-8e472909-06e2-4b40-b26c-d60bed2c4dc0](https://user-images.githubusercontent.com/125728453/233572973-4bf6a218-dc85-4b39-bc16-fcd683202ae3.jpg)

**Comment by oSumAtrIX** [comment]
You are using unofficial patches.

**Comment by PraveenAkio** [comment]
What is custom filters🫡🤷🏻‍♂️🙄

**Reply by hlytus** [reply]
https://www.reddit.com/r/revancedextended/comments/13dk8mp/how_to_use_custom_filters/
